pulsar/impl_partition_consumer.go

```diff
@@ -580,50 +610,52 @@ func (pc *partitionConsumer) HandlerMessage(response *pb.CommandMessage, headers
 	id := newMessageID(int64(msgID.GetLedgerId()), int64(msgID.GetEntryId()),
```

Comment:
Please change method name of `HandlerMessage` into "messageReceived" or "handleMessage"